src/GraphQL.Dynamic/LiteralGraphType/RemoteLiteralGraphType.cs

```diff
@@ -0,0 +1,344 @@
+            }
+
+            if (name == "String")
+            {
+                return LiteralGraphTypeMemberInfoType.String;
+            }
+
+            if (name == "Boolean")
+            {
+                return LiteralGraphTypeMemberInfoType.Boolean;
+            }
+
+            if (name == "Float")
+            {
+                return LiteralGraphTypeMemberInfoType.Float;
+            }
+
+            if (name == "ID")
+            {
+                return LiteralGraphTypeMemberInfoType.Guid;
+            }
+
+            if (name == "Date")
+            {
+                return LiteralGraphTypeMemberInfoType.DateTime;
+            }
+
+            if (name == "Decimal")
+            {
+                return LiteralGraphTypeMemberInfoType.Long;
+            }
+
+            return LiteralGraphTypeMemberInfoType.Unknown;
+        }
+
+        private bool TryGetFieldTypeFromRemoteSchema(Introspection.Schema schema, string name, out Introspection.TypeElement field)
+        {
+            field = schema.Types.FirstOrDefault(t => t.Name == name);
+
+            return field != null;
+        }
+
+        private static Introspection.Schema FetchRemoteServerSchema(string remote, RemoteSchemaFetcher remoteSchemaFetcher)
+        {
+            // lock types mutex
+            lock (FetchRemoteServerSchemaMutex)
+            {
+                // did someone else fetch it while we were waiting?
+                if (!RemoteServerSchemas.TryGetValue(remote, out var schema))
+                {
+                    //  fetch remote type information
+                    if (!TryGetRemoteSchema(remote, remoteSchemaFetcher, out schema))
+                    {
+                        // if it fails: fail
+                        // TODO: fail better
+                        throw new Exception("Failed to get remote schema");
+                    }
+
+                    // if it succeeds: save type information for all types
+                    RemoteServerSchemas.AddOrUpdate(remote, schema, (key, old) => schema);
+                }
+
+                return schema;
+            }
+        }
+
+        private static bool TryGetRemoteSchema(string remoteLocation, RemoteSchemaFetcher remoteSchemaFetcher, out Introspection.Schema schema)
+        {
+            schema = null;
+
+            try
+            {
+                schema = remoteSchemaFetcher(remoteLocation);
+            }
+            catch (Exception)
+            {
+                // TODO: log stuff
+            }
+
+            return schema != null;
+        }
+
+        private static Introspection.Schema FetchRemoteSchemaViaHttp(string url)
+        {
+            using (var client = new HttpClient())
+            {
+                var query = new
+                {
+                    Query = SchemaIntrospection.IntrospectionQuery
+                };
+
+                var response = client.PostAsync(new Uri(url), new StringContent(JsonConvert.SerializeObject(query)))
```

Comment:
as above with the accept header, setting the content type helped to solve an issue in our environment:
```
var response = client.PostAsync(new Uri(url), new StringContent(JsonConvert.SerializeObject(query), Encoding.UTF8, "application/json"))
```